src/Cms/ModelWithContent.php

```diff
@@ -637,6 +638,19 @@ public function uuid(): Uuid|null
 		return Uuid::for($this);
 	}
 
+	/**
+	 * Returns a new ID string if `content.uuid` option not disabled
+	 * @since 3.8.1
+	 */
+	public static function uuidGenerate(): string|null
```

Comment:
The method name isn't ideal in my opinion. I would go for something like Uuid::generateIfEnabled to make it clearer what's happening here. I was thinking first if it should actually be implemented directly in Uuid::generate instead, but I guess we could still need this even if uuids are disabled, right?

Reply:
> The method name isn't ideal in my opinion.

I was waiting for this comment 😅 I couldn't find a better name. I thought of a similar name `generateIfEnabled` but it felt like overkill 🙂 I'm open to all suggestions.

> I was thinking first if it should actually be implemented directly in Uuid::generate instead, but I guess we could still need this even if uuids are disabled, right?

Yes, this was discussed here https://github.com/getkirby/kirby/issues/4787#issuecomment-1279447162